spec/request/claims_spec.rb

```diff
@@ -8,4 +8,13 @@
       expect(response).to match_response_schema('claims')
     end
   end
+
+  it 'uploads documents supporting a claim' do
+    doctors_note = fixture_file_upload(
+      "#{::Rails.root}/spec/fixtures/files/doctors-note.pdf",
+      'application/pdf'
+    )
+    post '/v0/claims/189625/documents?tracked_item=33', file: doctors_note
+    expect(response).to be_success
+  end
```

Comment:
It still isn't creating a new cassette when I do that. I am very confused by it.